### PR TITLE
Migrate to maven move to org io.github.openfeign

### PR DESCRIPTION
**Pull request by spencergibb**
/cc @adriancole for your perusing pleasure :-)

### CODE REVIEW

**Comment by spencergibb** [comment]
Still missing any publishing framework, but we could adapt what is done for openzipkin.

**Comment by dsyer** [comment]
Why is it "feigntown"? (It's just hard to remember.)

**Comment by spencergibb** [comment]
@dsyer feign is taken. Open to suggestions. OpenFeign?

**Comment by spencergibb** [comment]
See #373 for adrian's suggestion.

**Comment by adriancole** [comment]
We should probably migrate this to the new org before merging this. before that, we need sonatype etc setup.

File: benchmark/src/main/java/feign/benchmark/RealRequestBenchmarks.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public void setup() {
     });
     server.start();
     client = new OkHttpClient();
-    client.setRetryOnConnectionFailure(false);
+// FIXME openfeign   client.setRetryOnConnectionFailure(false);
```
Comment (adriancole) [inline benchmark/src/main/java/feign/benchmark/RealRequestBenchmarks.java]: elaborate? :)

File: jackson-jaxb/pom.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  </developers>
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <properties>
+    <nebula_Manifest_Version>1.0</nebula_Manifest_Version>
```
Comment (adriancole) [inline jackson-jaxb/pom.xml]: wow.. we need all these things?

Comment (dsyer) [inline jackson-jaxb/pom.xml]: They could go in a parent pom. But you need them to comply with Sonatype's rules about Maven Central artifacts (which are not always enforced, but it's good citizens would do it).

Comment (adriancole) [inline jackson-jaxb/pom.xml]: I was referring to all the nebula stuff.. sorry should have indicated downwards

File: benchmark/src/main/java/feign/benchmark/RealRequestBenchmarks.java (hunk 1 above)
Comment (spencergibb) [inline benchmark/src/main/java/feign/benchmark/RealRequestBenchmarks.java]: `setRetryOnConnectionFailure` doesn't exist on `okhttp3.OkHttpClient`.

File: jackson-jaxb/pom.xml (hunk 2 above)
Comment (spencergibb) [inline jackson-jaxb/pom.xml]: Sorry, I missed one pom. I removed them all from the others.

**Comment by adriancole** [comment]
closed via #416 thanks for the help!

**Pull request closed without merging**